### PR TITLE
Disable sharing for public links and versions if app is disabled

### PR DESCRIPTION
**Pull request by juliushaertl**
- Do not pass the revision history parameter if files_versions is disabled
- Do not enable sharing for public links

### CODE REVIEW

File: src/helpers/url.js [hunk 1]
```diff
@@ -33,6 +33,9 @@ const getSearchParam = (name) => {
 }
 
 const getWopiUrl = ({ fileId, title, readOnly, closeButton, revisionHistory }) => {
+	// Only set the revision history parameter if the versions app is enabled
+	revisionHistory = revisionHistory && window?.oc_appswebroots?.files_versions
+
```
Comment (julien-nc) [inline src/helpers/url.js:38]: Is this a reliable way to check if an app is enabled? I mean are we sure it won't get removed anytime soon?

If not we could get the information via initial state I guess.

**Review by julien-nc — COMMENTED** [review]
Make sense!

Just a question.

File: src/helpers/url.js (hunk 1 above)
Comment (juliushaertl) [reply src/helpers/url.js:38]: Yeah, ideally we should have initial state for that, but I'd stick to this for now to keep it back portable and maybe pass over more stuff through initial state when getting back to #2258

Comment (julien-nc) [reply src/helpers/url.js:38]: :+1:

**Review by julien-nc — APPROVED** [review]

**Pull request merged**